package/MDAnalysis/analysis/hbonds/hbond_autocorrel.py

```diff
@@ -210,6 +266,10 @@ class HydrogenBondAutoCorrel(object):
         Within each run, the number of frames to analyse [50]
     pbc : bool, optional
         Whether to consider periodic boundaries in calculations [``True``]
+
+
+    .. versionchanged:: 0.20.0
+       If donors are not given, an attempt to automatically find these is done
```

Comment:
Add text explaining that `find_hydrogen_donors()` might be used, link to its docs.